bbcondeparser/html_tags.py

```diff
@@ -118,24 +121,12 @@ def validate_tag_cls(tag_cls):
         Raises:
             RuntimeError: if some values are set and shouldnt
         """
-        ctx_default = tag_cls.context_default
-        ctx_override = tag_cls.context_override
 
-        newline_behaviour = ctx_default.get('newline_behaviour')
-        if newline_behaviour not in NEWLINE_BEHAVIOURS:
+        if tag_cls.newline_behaviour not in NEWLINE_BEHAVIOURS:
             raise RuntimeError('newline_behaviour must be one of {}'.format(
```

Comment:
This should be a `ValueError`

This is my fault; I've used `RuntimeError` in other places and it's really not appropriate. Need to go back over the occurances of `RuntimeError` and switch them for more appropriate errors (probably `ValueError` or `TypeError`).